Project/Modules/Services/postService.cs

```diff
@@ -0,0 +1,86 @@
+using blog_net_core.Project.Modules.Model.Entities;
+using blog_net_core.Project.Modules.Model.Repositories;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace blog_net_core.Project.Modules.Services{
+
+    public class postService : IPostService{
+
+        private readonly PostsContext _context;
+
+        public postService(PostsContext context){
+
+                _context = context;
+        }
+
+            public  List<Posts> getAllPosts(){
```

Comment:
Transform this function as an asynchronous function.

Reply:
transformed to asynchronous task